pom.xml

```diff
@@ -832,6 +846,224 @@ Arrays, Maps or Strings. The iteration protocol was inspired by the Smalltalk co
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>spotless-check</id>
+
+            <properties>
+                <spotless.check.skip>false</spotless.check.skip>
+            </properties>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.diffplug.spotless</groupId>
+                        <artifactId>spotless-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>check</goal>
+                                </goals>
+                                <phase>validate</phase>
```

Comment:
The check goal is meant to be used in CI. It doesn't format code. It fails if the code isn't already formatted.